### PR TITLE
Fix/2377 fix permissions from gui

### LINKED ISSUE #2377: fixpermission from the webadmin
Add the ability to run `pfcmd fixpermissions` from the admin GUI

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description
Add the ability to "fix permissions" from the GUI

# Impacts
Admin GUI

# Issue
fixes #2377

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Ability to run "fixpermissions" from the web admin GUI

## Bug Fixes
* fixpermission from the webadmin (#2377)

### CODE REVIEW

File: lib/pf/util.pm [hunk 1]
```diff
@@ -502,6 +502,16 @@ sub fix_file_permissions {
     pf_run('sudo /usr/local/pf/bin/pfcmd fixpermissions file "' . $file . '"');
 }
 
+=item fix_files_permissions
```
Comment (jrouzierinverse) [inline lib/pf/util.pm:505]: Maybe a better name fix_permissions_all.
I find the name to close to the other function.

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]
After performing a fix permissions.

The fix permission option in the menu is no longer available unless I refresh the page.

The same behavior is found in perform checkup.

**Comment by julsemaan** [comment]
@jrouzierinverse, the checkup already had that behavior to prevent doing it 5 times if the user wanted to refer to it 5 different times given it takes a lot of time to accomplish.

IMO, I'd leave it to that behavior for all items of this menu but in the end you guys can decide together what is best

**Review by julsemaan — APPROVED** [review]

**Comment by jrouzierinverse** [comment]
@julsemaan As a policy you should clear a review unless you have done it.

**Comment by jrouzierinverse** [comment]
There was an additional issue there that was not addressed

**Comment by jrouzierinverse** [comment]
There should be at least a way to clear and rerun the results since some people would not know how rerun those commands.

**Comment by cgx** [comment]
I agree with @jrouzierinverse.

@dwlfrth: let me know if you need help.

**Comment by julsemaan** [comment]
@jrouzierinverse

I didn't clear your review, I just approved the PR since my review was requested on it yesterday

If you look, yours is still in place and left the PR in 'Changes requested'

**Comment by jrouzierinverse** [comment]
@julsemaan Sorry you are correct

**Comment by julsemaan** [comment]
@jrouzierinverse 

No problem, this gave me an idea for your next Christmas gift:
image

**Comment by jrouzierinverse** [comment]
Thanks!

**Comment by dwlfrth** [comment]
@jrouzierinverse maybe you could have a look at that additional issue (Option to have it automated when we detect a file permission issue.)

**Comment by dwlfrth** [comment]
GUI is adjusted.
@jrouzierinverse, are you taking care of the additional issue ?

**Pull request merged**